h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -693,11 +692,6 @@ private boolean exceededSearchLimits(String whatWeAreSkipping) {
       return true;
     }
 
-    if (timeRemainingMs() <= 0.001) {
```

Comment:
Do we need to remove this? Might be better not to start the modeling if we know for sure we will run out of time anyway

Reply:
I removed it bc I only want a single place of checking if time has run out. I can add it back, but the polling of the job should pick this up.